dietpi/dietpi-software

```diff
@@ -12604,6 +12660,19 @@ _EOF_
 			grep -v "minio" /DietPi/dietpi/.dietpi-services_include > /DietPi/dietpi/.dietpi-services_include.temp
 			mv /DietPi/dietpi/.dietpi-services_include.temp /DietPi/dietpi/.dietpi-services_include
 
+		elif (( $1 == 162)); then
+			# remove docker and all its unused dependencies
+			apt-get remove --auto-remove -y docker-engine
```

Comment:
You can use function ```AGP docker-engine``` here, it will purge.
```dietpi-software``` also runs ```apt-get autoremove --purge``` at end of uninstalls

Reply:
Excellent.